src/App.tsx

```diff
@@ -33,14 +34,16 @@ const App = () => {
   const [schema, setSchema] = useState<string>(() => initialAppData.schema)
   const [values, setValues] = useState<Array<string>>(() => initialAppData.values)
   const [version, setVersion] = useState(initialAppData.version)
+  const [packageName, setPackageName] = useState<ZodPackageName>(initialAppData.packageName)
```

Comment:
so the `packageName` wording sounds a bit alarming because in this way you could literally try to load every kind of package from the cdn by tweaking the localstorage (if I recall correctly, we store the initialappdata there).
I would have shaped the implementation by having a boolean value indicating the will of the user to use either `zod mini` or the normal one and then hardcode the two package names.